README.md

```diff
@@ -150,8 +150,8 @@ which has a `_start` entrypoint (e.g. a `src/main.rs` in Rust).
 
 [preview2-prototyping]: https://github.com/bytecodealliance/preview2-prototyping
 [preview1-build]: https://github.com/bytecodealliance/preview2-prototyping/releases/tag/latest
-[non-command]: https://github.com/bytecodealliance/preview2-prototyping/releases/download/latest/wasi_snapshot_preview1.reactor.wasm
-[command]: https://github.com/bytecodealliance/preview2-prototyping/releases/download/latest/wasi_snapshot_preview1.command.wasm
+[non-command]: https://github.com/bytecodealliance/preview2-prototyping/releases/download/latest/wasi_preview1_component_adapter.reactor.wasm
+[command]: https://github.com/bytecodealliance/preview2-prototyping/releases/download/latest/wasi_preview1_component_adapter.command.wasm
```

Comment:
```suggestion
[non-command]: https://github.com/bytecodealliance/wasmtime/releases/download/v10.0.1/wasi_snapshot_preview1.reactor.wasm
[command]: https://github.com/bytecodealliance/wasmtime/releases/download/v10.0.1/wasi_snapshot_preview1.command.wasm
```

Unfortunately, wasmtime doesn't have a "latest" tag, so we would have to update this on each wasmtime release, or we should open an issue on the wasmtime repo to add a latest tag.

Reply:
I believe that the `latest behavior is a github http redirect: `https://github.com/bytecodealliance/wasmtime/releases/latest` redirects to `https://github.com/bytecodealliance/wasmtime/releases/tag/v10.0.1` at the moment

Reply:
Thanks for the suggestions! I've updated this PR now with a commit that uses `latest` like @pchickey suggested.